src/UI/Component/Button/Tag.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/* Copyright (c) 2017 Nils Haagen <nils.haagen@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Button;
+
+use \ILIAS\UI\Component\Counter\Counter;
+
+/**
+ * This describes a tag(-button).
+ */
+interface Tag extends Button {
+	const RELLOW  		= 'btn-tag-relevance-verylow';
+	const RELVERYLOW 	= 'btn-tag-relevance-low';
+	const RELMID  		= 'btn-tag-relevance-middle';
+	const RELHIGH  		= 'btn-tag-relevance-high';
+	const RELVERYHIGH  	= 'btn-tag-relevance-veryhigh';
+
+	/**
+	 * Set relevance of Tag (to distinguis visually)
+	 *
+	 * @param	int	 $relevance  a value between 1 and 5
+	 * @throws 	\InvalidArgumentException
+	 * @return	Tag
+	 */
+	public function withRelevance($relevance);
+
+	/**
+	 * Get the relevance of the Tag.
+	 *
+	 * @return	int
+	 */
+	public function getRelevance();
+
+	/**
+	 * Get CSS-class according to the relevance of the Tag.
+	 *
+	 * @return	string
+	 */
+	public function getRelevanceClass();
+
+	/**
+	 * Set a fix background-color.
+	 *
+	 * @param	Color $col
+	 * @return	Tag
+	 */
+	public function withBackgroundColor($col);
```

Comment:
I also asked @Amstutz /Yvonne to try setup a palette with a maximum of what they think are distinguishable colors (for our cases like tags/calendar borders/letter avatars). After this point it just doesn't make any sense to add more colors.